js/app/state/preload.js

```diff
@@ -2,7 +2,7 @@
  * A module returning a function which will be executed to load game assets
  * @module app/state/preload
  */
-define(function(){
+define(['app/game'], function(game){
```

Comment:
Preload should not require 'game'. It is already passed to the preload callback (see the note [here](https://github.com/ALSchwalm/Quad/blob/master/js/app/game.js#L3))